web/src/main/java/org/springframework/security/web/jackson2/WebServletJackson2Module.java

```diff
@@ -56,6 +57,7 @@ public void setupModule(SetupContext context) {
 		context.setMixInAnnotations(SavedCookie.class, SavedCookieMixin.class);
 		context.setMixInAnnotations(DefaultSavedRequest.class, DefaultSavedRequestMixin.class);
 		context.setMixInAnnotations(WebAuthenticationDetails.class, WebAuthenticationDetailsMixin.class);
+		context.setMixInAnnotations(SwitchUserGrantedAuthority.class, SwitchUserGrantedAuthorityMixIn.class);
```

Comment:
Did we forget to set up this mixin for [WebServerJackson2Module](https://github.com/spring-projects/spring-security/blob/df640f22dcf63fdd1e54138cf4a358e268f7572f/web/src/main/java/org/springframework/security/web/server/jackson2/WebServerJackson2Module.java#L53-L56)?